pkg/agent/agent_windows.go

```diff
@@ -163,6 +165,10 @@ func (i *Initializer) prepareOVSBridge() error {
 			return err
 		}
 	}
+	if err := ovsCtlClient.SetPortNoFlood(config.UplinkOFPort); err != nil {
```

Comment:
Could you add some comments here explain why we do this?

Reply:
Sure, I will add some comments.